.cfnlintrc

```diff
@@ -8,4 +8,6 @@ ignore_checks:
   # E3012: Property Resources/EFSCname/Properties/TTL should be of type Long
   - E3012
   # E1001: Top level template section tests is not valid
-  - E1001
\ No newline at end of file
+  - E1001
+  # E9101: Checks that text is welcoming and inclusive as per Amazon Open Source Code of Conduct https://aws.github.io/code-of-conduct
```

Comment:
We might add one more line with a comment on why we are not doing this. Something along the lines
```
# Due to backward compatibility reasons we are not able to rename the parameters containing `Master` in their name as it could break existing deployments
```